### PR TITLE
Adding bmi_conversion.py

### PR DESCRIPTION
**Pull request by Snimer**
### **Describe your change:**
Adding bmi_conversion.py
It converts Mass and Height into Body Mass Index.
It contains two functions. One to convert BMI from Metric units and one from US units.

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: conversions/bmi_conversion.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    >>> metric_units_bmi(50, 160)
+    19.53
+    >>> metric_units_bmi(80, 175)
+    26.12
+    """
+    metre_square = (cm / 100) ** 2
+    return round(kg / metre_square, 2)
+
+
+if __name__ == "__main__":
```
Comment (mateuszz0000) [inline conversions/bmi_conversion.py:16]: Please remove duplicated `if __name__`

File: conversions/bmi_conversion.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    import doctest
+
+    doctest.testmod()
+
+
+def us_units_bmi(pounds: int, inches: int) -> int:
```
Comment (mateuszz0000) [inline conversions/bmi_conversion.py:22]: It will raise an exception if `inches==0`

Comment (mateuszz0000) [reply conversions/bmi_conversion.py:16]: Why resolved? You didn't fix it

Comment (Snimer) [reply conversions/bmi_conversion.py:16]: @mateuszz0000 
Please check again. 
From my side I have fixed it. I have also viewed the file inside this pull request. There is now only one "if main" on line 28. 

If this is not the case please tell me where I am wrong.

**Review by 53jk1 — APPROVED** [review]

**Review by mateuszz0000 — CHANGES_REQUESTED** [review]
Please add following doctest to `us_units_bmi`:
```
>>> us_units_bmi(130, 0)
  WHAT RESULT WE SHOULD EXPECT?
```

**Comment by Snimer** [comment]
> Please add following doctest to `us_units_bmi`:
> 
> ```
> >>> us_units_bmi(130, 0)
>   WHAT RESULT WE SHOULD EXPECT?
> ```
@mateuszz0000 
I am closing this pull request because I deleted the branch from where the file was.  
So, I am made a different pull request. Please review this one- 
#3727

**Pull request closed without merging**